src/Workspaces/Core/Portable/FindSymbols/Shared/AbstractSyntaxIndex_Persistence.cs

```diff
@@ -17,7 +17,7 @@ namespace Microsoft.CodeAnalysis.FindSymbols
     internal partial class AbstractSyntaxIndex<TIndex>
     {
         private static readonly string s_persistenceName = typeof(TIndex).Name;
-        private static readonly Checksum s_serializationFormatChecksum = CodeAnalysis.Checksum.Create("38");
+        private static readonly Checksum s_serializationFormatChecksum = CodeAnalysis.Checksum.Create("39");
```

Comment:
The format of BKTree changed, so needed to rev this.

Reply:
Does the format change?

Reply:
Oh nevermind, it's the persist format changes. Make sense.